packages/ui5-middleware-fe-mockserver/README.md

```diff
@@ -77,16 +72,16 @@ Each service must provide at least two things
 On top of that you can specify one of the following option
 
 - mockdataPath : the path to the folder containing the mockdata files
-- generateMockData : whether or not you want to use automatically generated mockdata
+- generateMockData : whether or not you want to use automatically generated mockdata <-- TODO: what happens if i set this to false and only define 1 of 2 entites in the service? error in the frontend?
 - forceNullableValuesToNull: determine if nullable properties should be generated as null or with a default value (defaults to false)
 
 Additional option are available either per service of for all services if defined globally
 
 - debug : toggle the debug mode
 - watch : toggle the watch mode, the mockserver will restart the service where data or metadata have changed
 - noETag : disable ETag support on metadata
-- strictKeyMode : disable the default "loose" mode for the key matching, you can try this if the mockserver returns too much data
-- contextBasedIsolation : enable the support of "tenants", by adding /tenant-xxx at the very start of your service call you will be able to work on tenant isolated data.
+- strictKeyMode : disable the default "loose" mode for the key matching, you can try this if the mockserver returns too much data <-- TODO: maybe a little bit more background info about what "loose" means here
```

Comment:
```suggestion
- strictKeyMode : disable the default "loose" mode for the key matching.
By default the mockserver will try many ways to match an entityset with its navigation property, which can result in some rather loose join of data. In case you have a very well defined mockdata and you feel the mockserver is returning too much data, you can enable this feature.
```